src/s1reader/s1_burst_slc.py

```diff
@@ -498,7 +506,8 @@ def geometrical_and_steering_doppler(self, xstep=500, ystep=50):
 
         return x, y, total_doppler
```

Comment:
Don't we want to return this as a `isce3.core.LUT2d`? My understanding was that the `isce3.geocode.geocodeSlc `module is accepting the corrections as LUTs and not as `numpy` arrays.